MoveNet_Vs_PoseNet.ipynb

```diff
@@ -463,7 +460,7 @@
       "\n",
       "         LShoulder_score  RShoulder_score  LHip_score  RHip_score\n",
       "movenet         0.729215          0.58562    0.488324    0.618876\n",
-      "posenet         0.693424          0.50000    0.746721    0.500000\n",
+      "posenet         0.693424          0.00000    0.746721    0.000000\n",
```

Comment:
@bhavikapanara why are these values hardcoded instead of being dynamically calculated?